templates/common/bin/get_net_ip

```diff
@@ -40,7 +42,19 @@ import sys
 
 def get_ip_from_network_status():
     namespace = os.environ.get('PodNamespace')
-    net_status = json.loads(os.environ.get('PodNetworksStatus'))
+    try:
+        for attempt in Retrying(
+            stop=stop_after_attempt(5),
+            wait=wait_exponential(multiplier=1, min=2, max=10)):
+            with attempt:
+                with open('/etc/podinfo/network-status') as f:
+                    network_status_content = f.read()
+                    net_status = json.loads(network_status_content)
+    except RetryError:
+        raise Exception(
+            f"Unable to load pod network_status - /etc/podinfo/network-status contains: "
+            f"{network_status_content}")
```

Comment:
in theory, you may reach this line without network_status_content ever declared (e.g. if open() always fails, which will raise NameError. Maybe initialize the variable before try? (None will do)

Reply:
Agreed, otherwise looks good

Reply:
Good catch.

Done!